apps/chrome-devtools/src/app-devtools/theming-panel/theming-panel-pres.template.html

```diff
@@ -28,7 +28,7 @@
           <h3 ngbAccordionHeader>
             <button ngbAccordionButton>
               @if (group.isPalette && group.defaultVariable) {
-                <div (click)="$event.stopPropagation();">
+                <div tabindex="-1" (click)="$event.stopPropagation();" (keyup.enter)="$event.stopPropagation();">
```

Comment:
What the $event.stopPropagation for here? The button is not a type=submit, is there really an event to stop propagating? (Actually why is the button on the whole html block and not around the label?)

Reply:
I added a comment above to explain why we need to stop the propagation